indexer/services/ender/src/validators/stateful-order-validator.ts

```
@@ -189,9 +199,40 @@ export class StatefulOrderValidator extends Validator<StatefulOrderEventV1> {
 
     this.validateStatefulOrder(order);
 
-    if (order.orderId!.orderFlags !== ORDER_FLAG_LONG_TERM) {
+    if (order.orderId!.orderFlags !== ORDER_FLAG_LONG_TERM &&
+      order.orderId!.orderFlags !== ORDER_FLAG_TWAP_SUBORDER) {
       return this.logAndThrowParseMessageError(
-        `StatefulOrderEvent long term order must have order flag ${ORDER_FLAG_LONG_TERM}`,
+        'StatefulOrderEvent long term order must have order flag ' +
+        `${ORDER_FLAG_LONG_TERM} or ${ORDER_FLAG_TWAP_SUBORDER}`,
+        { event: this.event },
+      );
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)